src/wymeditor/editor/base.js

```diff
@@ -3018,7 +3018,10 @@ WYMeditor.editor.prototype._getSelectedNodes = function () {
         return false;
     }
 
-    selectedNodes = selection.getRangeAt(0).getNodes();
+    selectedNodes = selection.getAllRanges()
+        .reduce(function(nodes, range) {
+            return nodes.concat(range.getNodes());
+        }, [])
```

Comment:
Firefox now returns a selection with three ranges, one for each cell.

This new code works in all browsers.

It is funny code, for me. I had no trouble writing it and it was fun. But, I don't like coming across code like this. It seems hard to read. I know that for others it may seem very easy to read. Is this a matter of experience?